server/bin/gold/test-24.txt

```diff
@@ -1,230 +0,0 @@
-+++ Verifying server activation
```

Comment:
Note that `test-24` was removed because it dealt with activating 2 versions of the prep shim at the same time.  We just have one now, the merged behavior into `pbench-dispatch.sh`.